src/new-client/src/App/TearOutSection/tearOutContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { createContext } from "react"
+
+export const sectionsTornOut = {
+  isTornOut: false,
+}
+
+export const TearOutContext = createContext(sectionsTornOut)
```

Comment:
Can you add an explicit type here, it's a good practice to get into.

Reply:
Done